oak_marketplace_emotion_recognition/utils.py

```diff
@@ -0,0 +1,235 @@
+                               np.clip(result_statistic.shape[1] - self.graphs_size - Y_OFFSET, 0, result_statistic.shape[0])
+        for emotion, ellipse_graph in self.graphs.items():
+            if emotion == top_emotion:
+                continue
+            percent = self.get_current_emotion_percent(emotion)
+            text = f'{emotion.upper()} - {percent}%'
+            result_statistic = place_class_names_and_percents(result_statistic, [start_x, int(result_statistic.shape[1] - Y_OFFSET / 2)],
+                                                        text)
+            alpha_mask = ellipse_graph.alpha_mask
+            overlay_image_alpha(result_statistic, ellipse_graph.graph, start_x, start_y, alpha_mask)
+            start_x = np.clip(start_x - self.graphs_size - 15 * X_OFFSET, 0, result_statistic.shape[0])
+        if is_save:
+            cv2.imwrite('result_statistic.png', result_statistic)
+        return result_statistic
+
+
+class EmotionGraph:
+    def __init__(
+        self, background_color: tuple = (28, 28, 30),
+        progress_color: tuple = (145, 82, 225),
+        size: int = 80,
+        thickness: int = 10,
+        start_angle: int = -220,
+        end_angle: int = 40,
+        emotion: str = 'happy',
+        emotion_size: tuple = (25, 25)
+    ):
+        self.background_color = background_color
+        self.progress_color = progress_color
+        self.size = size
+        self.thickness = thickness
+        self.start_angle = start_angle
+        self.end_angle = end_angle
+        self.emotion_size = emotion_size
+        self.emotion_images = {
+            'neutral': np.array(
+                Image.open(
+                    os.path.join(
+                        os.path.dirname(__file__),
+                        "emotions-pack", "neutral.png",
+                    ),
+                ).resize(self.emotion_size),
+            ),
+            'happy': np.array(
+                Image.open(
+                    os.path.join(
+                        os.path.dirname(__file__),
+                        "emotions-pack", "happy.png",
+                    ),
+                ).resize(self.emotion_size),
+            ),
+            'sad': np.array(
+                Image.open(
+                    os.path.join(
+                        os.path.dirname(__file__),
+                        "emotions-pack", "sad.png",
+                    ),
+                ).resize(self.emotion_size),
+            ),
+            'anger': np.array(
+                Image.open(
+                    os.path.join(
+                        os.path.dirname(__file__),
+                        "emotions-pack", "anger.png",
+                    ),
+                ).resize(self.emotion_size),
+            ),
+            'surprise': np.array(
+                Image.open(
+                    os.path.join(
+                        os.path.dirname(__file__),
+                        "emotions-pack", "surprise.png",
```

Comment:
Don't hard-code paths. Please, add emotions-path: "images/emoji" to init function